week-07/day-4/HelloWorld/HelloWorld/Models/StudentService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace HelloWorld.Models
+{
+    public class StudentService
+    {
+        private readonly List<string> names;
+        private int count;
+
+        public StudentService()
+        {
+            names = new List<string> { "Sanyi", "Lilla", "John" };
+        }
+
+        public List<string> FindAll()
+        {
+            return names;
+        }
+
+        public void Save(string student)
+        {
+            names.Add(student);
+        }
+
+        public int Count()
+        {
+            count = names.Count;
```

Comment:
why is the `count` saved like that? why not always return `names.Count` directly?